frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts

```diff
@@ -1,6 +1,6 @@
 export const sideBarTabs = {
   details: 'Details',
   resources: 'Resources',
-  releaseNotes: 'Release notes',
+  releaseNotes: 'Release Notes',
```

Comment:
I think we don't need this change. We follow this casing convention everywhere in console.

Reply:
@rohitkrai03 some of the cypress tests were failing because of this text because on the Side Bar tabs we have `'Release Notes'`.

Reply:
I think he means in tab as well we should show `Release notes` as text

Reply:
Yeah, I meant the tab was originally named `Release notes`, right?